src/admin/python/cli/entrypoint.py

```diff
@@ -250,21 +250,6 @@ def replication_controller(**kwargs):
     script.enter_replication_controller(**kwargs)
 
 
-@entrypoint.command()
-@click.option(
-    "--dashboard-port",
-    help="The port the dashboard will serve on.",
-)
-@click.option(
-    "--dashboard-html",
-    help="The path to the folder with the html sources for the nginx dashboard.",
-)
-@repl_ctrl_domain_name_option()
-@repl_ctrl_port_option()
```

Comment:
it appears this option can be deleted from options.py, I think this is the only use of it. (repl_ctrl_domain_name_option *is* used in other places tho)

Reply:
Hmm -- don't see it listed in options.py?

Reply:
I _do_ see it on the host side cli, but I think it is still wanted there -- it is plumbed to the `port` directive in compose.yaml, and still necessary as a default override for users to avoid port collisions in shared environments like `master02`.